# Focal length x.
float64 fx
# Focal length y.
float64 fy
# Principal point x.
float64 cx
# Principal point y.
float64 cy
